package.json

```diff
@@ -50,6 +50,7 @@
     "nunjucks": "^3.0.1",
     "postcodesio-client": "billinghamj/postcodesio-client-node",
     "require-environment-variables": "^1.1.2",
+    "tunnel-agent": "^0.6.0",
```

Comment:
Upgrading sub-dependencies is not a thing in yarn yet (https://github.com/yarnpkg/yarn/issues/2394) so this is a valid workaround of manually upgrading the sub-dependency of node-sass to be able to pass snyk.